src/core/createElement.js

```diff
@@ -61,7 +64,7 @@ export function createChildren(children) {
 
 		for (let i = 0; i < children.length; i++) {
 			const child = children[i];
-			if (!isNullOrUndefined(child)) {
+			if (!isNullOrUndefined(child) && typeof child === 'object') {
```

Comment:
This check might not be strong enough, since we still pass already constructed children to `createChild`.